blog/2023-12-22-openhab-4-1-release.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: BlogPost
+category: blog
+title: openHAB 4.1 Release
+author: Florian Hotze and others
+date: '2023-12-22T18:00:00+02:00'
+previewimage: /uploads/2023-12-22-openhab-4-1-release/header.jpg
+tags:
+  - news
+excerpt: >-
+  As usual before christmas, we are happy to announce a new release of openHAB, that brings many additions, improvements and fixes!
+---
+The first minor release of openHAB 4 has arrived! openHAB 4.1 brings several improvements and bugfixes as well as some nice additions!
+
+TODO: Write intro
+
+With that being said, we again want to share some statistics that show the activity in numbers.
+
+# Activity
+
+As you'd expect from any new openHAB release, we have had a high activity across all our GitHub repositories since our [summer release](/blog/2023-07-23-openhab-4-0-release.html).
+We therefore want to share a few statistics about our three main repositories: Core, add-ons and UI.
+
+[openHAB core](https://github.com/openhab/openhab-core) has received 122 pull requests in total, with 31 bug fixes and 46 enhancements resulting in 10,238 lines of code added.
+Even though the number of pull requests decreased compared to our [summer release](/blog/2023-07-23-openhab-4-0-release.html), that is still a lot of activity.
+
+5 new add-ons were introduced, and with 437 pull requests, 121 bug fixes and 132 enhancements resulted in 13,071 lines of add-on code added to the [openHAB add-ons repository](https://github.com/openhab/openhab-addons).
+
+Our [openHAB web UIs](https://github.com/openhab/openhab-webui) have also received many contributions: 129 pull requests including 46 bug fixes and 67 enhancements resulted in 6,235 lines of code changed.
+This means the number of pull requests only slightly decreased compared to our [summer release](/blog/2023-07-23-openhab-4-0-release.html), which means our UIs continue to develop at a high pace.
```

Comment:
Again i would not mention its less, just that it continues to be actively developed.